spec/game_spec.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../classes/game'
+
+RSpec.describe Game do
+  let(:publish_date) { Date.new(2022, 1, 1) }
+  let(:last_played_at) { DateTime.new(2023, 1, 1) }
+  let(:game) { Game.new(true, last_played_at, publish_date) }
+
+  describe '#initialize' do
+    it 'creates a new game with specified attributes' do
+      expect(game.multiplayer).to eq(true)
+      expect(game.last_played_at).to eq(last_played_at)
+      expect(game.publish_date).to eq(publish_date)
+      expect(game.archived).to eq(false) # Default value for archived is false
+    end
+  end
+
+  describe '#can_be_archived?' do
+    context 'when last_played_at is more than 2 years ago' do
+      let(:last_played_at) { DateTime.new(2020, 1, 1) }
+
+      it 'returns true' do
+        expect(game.can_be_archived?).to eq(true)
+      end
+    end
+
```

Comment:
Kindly add a new test case after line 24 when `super` is false and `last_played_at` if more than 2:
```
context 'when last_played_at is more than 2 years ago and super is false' do
  let(:last_played_at) { '2020-01-01' }
  let(:publish_date) { '2022-01-01' }

  it 'returns true' do
    expect(game.can_be_archived?).to eq(false)
  end
end
```